### ISSUE TITLE
code 500错误

### ISSUE THREAD

**Issue by cj1357** [oldweipro/claude-to-chatgpt#22]
![image](https://github.com/oldweipro/claude-to-chatgpt/assets/40661342/097b85b5-810d-430c-b0d5-a9c68cce43a9)
![image](https://github.com/oldweipro/claude-to-chatgpt/assets/40661342/c39fbedb-59cc-444b-9ec5-cf9f00d7e2c2)

**Comment by cj1357** [comment]
造成这个错误的原因是模型名称变化了，我修改后测试已经没问题了

参考了https://github.com/KoushikNavuluri/Claude-API/issues/85 
需要把请求的模型名称改成"claude-2.1"
[claude.go 56行](https://github.com/oldweipro/claude-to-chatgpt/blob/182f543586cee4b4688ae6e21ce90df9cc31f212/model/claude.go#L56)

**Comment by cj1357** [comment]
![image](https://github.com/oldweipro/claude-to-chatgpt/assets/40661342/f7617a81-2366-40d6-88ba-34db20ac64c4)
不过用chatgpt-next-web调用，返回的格式会带身份信息，这个如果能给优化一下就好了😺

**Comment by spammenotinoz** [comment]
> The reason for this error is that the model name has changed, and I have modified the test to be fine
> 
> Reference [KoushikNavuluri/Claude-API #85](https://github.com/KoushikNavuluri/Claude-API/issues/85) Need to change the model name of the request to "claude-2.1" [claude.go line 56](https://github.com/oldweipro/claude-to-chatgpt/blob/182f543586cee4b4688ae6e21ce90df9cc31f212/model/claude.go#L56)

Thanks this works.

**Comment by cj1357** [comment]
模型名称又发生了改变，需要换成"claude-2.0"
https://github.com/oldweipro/claude-to-chatgpt/blob/182f543586cee4b4688ae6e21ce90df9cc31f212/model/claude.go#L56
注意不是原先的claude-2

参考了https://github.com/KoushikNavuluri/Claude-API/issues/91

**Comment by caicaicaibeiqie** [comment]
我在使用claude_api的send_message时遇到了同样的错误返回，更新了model问题依然存在，还有什么可能的参数问题吗？

**Comment by cj1357** [comment]
> 我在使用claude_api的send_message时遇到了同样的错误返回，更新了model问题依然存在，还有什么可能的参数问题吗？

好像官方那边又更新了策略，改模型名称解决不了了，不知道怎么解决

**Comment by caicaicaibeiqie** [comment]
@cj1357 You are right, but I still need to decode the "headers" parameter of "send_message". Problem solved.
